File: Domain/Book.cs
using System;

namespace Domain
{
    public class Book
    {
        public Guid Id { get; set; }
        
        public string BookTitle { get; set; }

        public string BookSubtitle { get; set; }

        public string Author { get; set; }

        public string Thumbnail { get; set; }

        public string Notes { get; set; }

        public string BookUrl { get; set; }

        public DateTime Date { get; set; }

        public string UserName { get; set; }

        public string Status { get; set; }

        public bool Favorite { get; set; }

    }
}

File: react_finder/src/models/book.ts
interface Book {
    id: string,
    bookTitle: string,
    bookSubtitle: string,
    author: string,
    thumbnail: string,
    notes: string,
    bookUrl: string,
    date: Date | null,
    userName: string,
    status: string,
    favorite: boolean
}
// to add subtitle, author, thumbnail, and myNotes
export default Book;
